runtime/src/bank.rs

```diff
@@ -4751,10 +4790,9 @@ impl Bank {
             ) => programdata_account
                 .data()
                 .get(UpgradeableLoaderState::size_of_programdata_metadata()..)
-                .ok_or(InstructionError::InvalidAccountData)
+                .ok_or("Invalid account data".into())
```

Comment:
why is this a string?

Reply:
Just needed a placeholder to match the error type returned by `Self::load_program_from_bytes()`.

Reply:
It returns a `Box<dyn std::error::Error>`, we could `Box` `InstructionError::InvalidAccountData` here as well.